.github/workflows/ci-emscripten.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       uses: actions/cache@v4
       with:
         path: /emsdk/upstream/emscripten/cache
-        key: 3.1.57-${{ runner.os }}
+        key: 4.0.3-${{ runner.os }}
```

Comment:
If this fixes your issue, also be mindful to update https://github.com/OpenTTD/OpenTTD/blob/master/os/emscripten/Dockerfile :)